<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Haky Im">
<meta name="dcterms.date" content="2024-03-20">

<title>HKI Courses - Dissection of Genetics &amp; Prediction of Cancer Risk: Methods &amp; Tools</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../../">
<script src="../../../site_libs/quarto-html/quarto.js"></script>
<script src="../../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="../../../styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../../../index.html">
    <span class="navbar-title">HKI Courses</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../../index.html" rel="" target="">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../../index-bios-25328.html" rel="" target="">
 <span class="menu-text">BIOS25328</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../../index-logistics.html" rel="" target="">
 <span class="menu-text">Logistics</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#goals-of-the-class" id="toc-goals-of-the-class" class="nav-link active" data-scroll-target="#goals-of-the-class">Goals of the Class</a></li>
  <li><a href="#linkage-analysis" id="toc-linkage-analysis" class="nav-link" data-scroll-target="#linkage-analysis">Linkage Analysis</a></li>
  <li><a href="#high-penetrance-cancer-susceptibility-genes" id="toc-high-penetrance-cancer-susceptibility-genes" class="nav-link" data-scroll-target="#high-penetrance-cancer-susceptibility-genes">High Penetrance Cancer Susceptibility Genes</a></li>
  <li><a href="#power-of-association-vs-linkage" id="toc-power-of-association-vs-linkage" class="nav-link" data-scroll-target="#power-of-association-vs-linkage">1996 - Power of Association vs Linkage</a></li>
  <li><a href="#gene-mapping-methods" id="toc-gene-mapping-methods" class="nav-link" data-scroll-target="#gene-mapping-methods">Gene Mapping Methods</a></li>
  <li><a href="#expected-to-find-high-penetrance-genes" id="toc-expected-to-find-high-penetrance-genes" class="nav-link" data-scroll-target="#expected-to-find-high-penetrance-genes">Expected to Find High Penetrance Genes</a></li>
  <li><a href="#found-many-low-penetrance-variants" id="toc-found-many-low-penetrance-variants" class="nav-link" data-scroll-target="#found-many-low-penetrance-variants">Found Many Low Penetrance Variants</a></li>
  <li><a href="#effect-size-vs.-allele-frequency" id="toc-effect-size-vs.-allele-frequency" class="nav-link" data-scroll-target="#effect-size-vs.-allele-frequency">Effect Size vs.&nbsp;Allele Frequency</a></li>
  <li><a href="#review-hypothesis-testing" id="toc-review-hypothesis-testing" class="nav-link" data-scroll-target="#review-hypothesis-testing">Review: Hypothesis Testing</a></li>
  <li><a href="#review-hypothesis-testing-cont." id="toc-review-hypothesis-testing-cont." class="nav-link" data-scroll-target="#review-hypothesis-testing-cont.">Review: Hypothesis Testing (Cont.)</a></li>
  <li><a href="#regression-approach-single-snp" id="toc-regression-approach-single-snp" class="nav-link" data-scroll-target="#regression-approach-single-snp">Regression Approach: Single SNP</a></li>
  <li><a href="#regression-approach-for-quantitative-traits" id="toc-regression-approach-for-quantitative-traits" class="nav-link" data-scroll-target="#regression-approach-for-quantitative-traits">Regression Approach for Quantitative Traits</a></li>
  <li><a href="#regression-approach-for-disease-traits" id="toc-regression-approach-for-disease-traits" class="nav-link" data-scroll-target="#regression-approach-for-disease-traits">Regression Approach for Disease Traits</a></li>
  <li><a href="#genome-wide-association-studies-gwas" id="toc-genome-wide-association-studies-gwas" class="nav-link" data-scroll-target="#genome-wide-association-studies-gwas">Genome-wide Association Studies (GWAS)</a></li>
  <li><a href="#gwas-genotype-phenotype" id="toc-gwas-genotype-phenotype" class="nav-link" data-scroll-target="#gwas-genotype-phenotype">GWAS: Genotype + Phenotype</a></li>
  <li><a href="#manhattan-plot" id="toc-manhattan-plot" class="nav-link" data-scroll-target="#manhattan-plot">Manhattan Plot</a></li>
  <li><a href="#qq-plot" id="toc-qq-plot" class="nav-link" data-scroll-target="#qq-plot">QQ Plot</a></li>
  <li><a href="#gwas-snp-trait-discovery-timeline" id="toc-gwas-snp-trait-discovery-timeline" class="nav-link" data-scroll-target="#gwas-snp-trait-discovery-timeline">GWAS SNP-Trait Discovery Timeline</a></li>
  <li><a href="#wtccc-first-large-scale-gwas" id="toc-wtccc-first-large-scale-gwas" class="nav-link" data-scroll-target="#wtccc-first-large-scale-gwas">WTCCC: First Large Scale GWAS</a></li>
  <li><a href="#wtccc-population-structure" id="toc-wtccc-population-structure" class="nav-link" data-scroll-target="#wtccc-population-structure">WTCCC: Population Structure</a></li>
  <li><a href="#zoom-in-with-regional-plots" id="toc-zoom-in-with-regional-plots" class="nav-link" data-scroll-target="#zoom-in-with-regional-plots">Zoom in with Regional Plots</a></li>
  <li><a href="#clarifications-and-logistics" id="toc-clarifications-and-logistics" class="nav-link" data-scroll-target="#clarifications-and-logistics">Clarifications and Logistics</a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="index.html"><i class="bi bi-file-slides"></i>RevealJS</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<div class="quarto-title-block"><div><h1 class="title">Dissection of Genetics &amp; Prediction of Cancer Risk: Methods &amp; Tools</h1><button type="button" class="btn code-tools-button" id="quarto-code-tools-source"><i class="bi"></i> Code</button></div></div>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Haky Im </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">March 20, 2024</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<section id="goals-of-the-class" class="level2">
<h2 class="anchored" data-anchor-id="goals-of-the-class">Goals of the Class</h2>
<ul>
<li>Learn concepts and tools needed to understand the current literature in cancer genomics and interpret the results</li>
<li>First month: focus on germline risk factors</li>
<li>Today:
<ul>
<li>Gene mapping: Linkage vs.&nbsp;association approaches</li>
<li>Describe the genetic architecture of cancer risk</li>
<li>Discuss Genome-wide Association Studies (GWAS)</li>
</ul></li>
</ul>
<hr>
</section>
<section id="linkage-analysis" class="level2">
<h2 class="anchored" data-anchor-id="linkage-analysis">Linkage Analysis</h2>
<ul>
<li>Gene mapping method popular before development of high throughput genotyping techniques</li>
<li>100’s of markers interrogated (causal markers most likely missed)</li>
<li>Important genes were found with this approach
<ul>
<li>CFTR: Cystic Fibrosis</li>
<li>HTT: Huntington disease</li>
<li>NOD2: inflammatory bowel disease</li>
</ul></li>
<li>Markers that are co-transmitted (recombination rate&lt;0.5) with the disease susceptibility locus (DSL) help locate the DSL</li>
<li>It uses families with many cases (“loaded pedigrees”)</li>
</ul>
<hr>
</section>
<section id="high-penetrance-cancer-susceptibility-genes" class="level2">
<h2 class="anchored" data-anchor-id="high-penetrance-cancer-susceptibility-genes">High Penetrance Cancer Susceptibility Genes</h2>
<ul>
<li>Genetic linkage performed in the 1980s and 1990s found:
<ul>
<li>Breast and ovarian cancers
<ul>
<li>BRCA1</li>
<li>BRCA2</li>
</ul></li>
<li>Colorectal cancer
<ul>
<li>adenomatous polyposis coli (APC)</li>
<li>mismatch repair genes</li>
<li>mutL homologue 1 (MLH1)</li>
<li>mutS homologue 2 (MSH2)</li>
</ul></li>
<li>Melanoma: cyclin-dependent kinase inhibitor 2A (CDKN2A)</li>
</ul></li>
<li>To date mutations in more than 70 CSGs associated with high-penetrance cancer susceptibility syndromes have been identified with RRs of 5-100</li>
</ul>
<hr>
</section>
<section id="power-of-association-vs-linkage" class="level2">
<h2 class="anchored" data-anchor-id="power-of-association-vs-linkage">1996 - Power of Association vs Linkage</h2>
<ul>
<li>Risch N. &amp; Merikangas K. The future of genetic studies of complex human diseases. Science 273, 1516–1517 (1996).</li>
<li>Many high penetrance (large effect) cancer susceptibility genes were found using linkage analysis.</li>
<li>By mid 90’s discoveries using linkage were dwindling prompting the influential paper by Neil Risch and Merikangas to propose a revolutionary idea of genotyping about 100K loci in the genome (considered in the realm of science fiction at the time).</li>
<li>Technological advances brought about by the human genome project made genome-wide association studies possible.</li>
</ul>
<hr>
</section>
<section id="gene-mapping-methods" class="level2">
<h2 class="anchored" data-anchor-id="gene-mapping-methods">Gene Mapping Methods</h2>
<ul>
<li><strong>Linkage analysis</strong>
<ul>
<li>popular before high throughput technology became available</li>
<li>based on co-transmission of genetic markers and disease genes</li>
<li>a few hundred markers can cover the whole genome</li>
<li>low resolution</li>
<li>pedigree/family based</li>
</ul></li>
<li><strong>Association mapping</strong>
<ul>
<li>need a large number of markers (&gt; 1 million)</li>
<li>genetic markers in LD (Linkage Disequilibrium) with disease genes</li>
<li>higher resolution</li>
<li>families not needed (easier to recruit large cohorts)</li>
</ul></li>
</ul>
<hr>
</section>
<section id="expected-to-find-high-penetrance-genes" class="level2">
<h2 class="anchored" data-anchor-id="expected-to-find-high-penetrance-genes">Expected to Find High Penetrance Genes</h2>
<ul>
<li>Recap:
<ul>
<li>Linkage and association are main approaches for mapping genes to diseases and other human traits.</li>
<li>Before the completion of the Human Genome Project, linkage analysis was the most popular approach to identify disease loci.</li>
<li>Association methods are based on the LD between genetic markers and disease genes.</li>
<li>We’ve been performing genome-wide association studies for the last two decades, expecting to find high penetrance (large effect size) genes.</li>
</ul></li>
</ul>
<hr>
</section>
<section id="found-many-low-penetrance-variants" class="level2">
<h2 class="anchored" data-anchor-id="found-many-low-penetrance-variants">Found Many Low Penetrance Variants</h2>
<ul>
<li>Instead, we found a large number of variants with modest effect sizes (low penetrance).</li>
<li>We found that the genetic architecture of cancer risk was highly polygenic.</li>
<li>This requires better statistical methods to make sense of the discoveries.</li>
</ul>
<hr>
</section>
<section id="effect-size-vs.-allele-frequency" class="level2">
<h2 class="anchored" data-anchor-id="effect-size-vs.-allele-frequency">Effect Size vs.&nbsp;Allele Frequency</h2>
<ul>
<li>The figure shows that the discoverability of the risk alleles depends on the population frequency.</li>
<li>GWAS have unlocked many discoveries on the lower left, but to make sense of them and turn them into actionable targets, we need to learn new statistical concepts.</li>
<li>Low penetrance = low relative risk = small effect size</li>
</ul>
<hr>
</section>
<section id="review-hypothesis-testing" class="level2">
<h2 class="anchored" data-anchor-id="review-hypothesis-testing">Review: Hypothesis Testing</h2>
<ul>
<li><strong>H0</strong>: null hypothesis (e.g., no difference in case-control allele frequencies)</li>
<li><strong>HA</strong>: alternative hypothesis (there is a difference, i.e., correlated with the causal variant)</li>
<li><strong>Test statistic Z</strong>: in many situations associated with a model</li>
<li><strong>Significance level (α)</strong>: allowed type I error (reject null when null is true)</li>
<li><strong>p-value</strong>: P(observed test statistic more extreme than the threshold | H0)</li>
<li>We reject the null hypothesis if the observed test statistic is more extreme than what we would expect could happen by chance.</li>
</ul>
<hr>
</section>
<section id="review-hypothesis-testing-cont." class="level2">
<h2 class="anchored" data-anchor-id="review-hypothesis-testing-cont.">Review: Hypothesis Testing (Cont.)</h2>
<ul>
<li><strong>Type I error (α)</strong>: probability of rejecting the null when the null is true</li>
<li><strong>Type II error (β)</strong>: probability of not rejecting the null when the null is false</li>
<li><strong>Power</strong>: probability that we will reject the null when the alternative is true</li>
<li><strong>P-value</strong>: probability that we will get a result as extreme or more extreme than the observed data point under the null hypothesis</li>
</ul>
<hr>
</section>
<section id="regression-approach-single-snp" class="level2">
<h2 class="anchored" data-anchor-id="regression-approach-single-snp">Regression Approach: Single SNP</h2>
<ul>
<li>SNP: single nucleotide polymorphism</li>
<li><strong>Regression model</strong>: Y = μ + a ⋅ age + β ⋅ X + ε
<ul>
<li>Parameters β are estimated (using MLE, least squares, etc.)</li>
<li>Null hypothesis: β = 0</li>
<li>Many types of traits can be treated with the same approach</li>
<li>Can correct for covariates (age, sex, ancestry)</li>
<li>Prediction</li>
</ul></li>
<li><strong>Genotype</strong>: aa, aA, AA</li>
<li><strong>Dosage (X1)</strong>: number of A alleles</li>
</ul>
<hr>
</section>
<section id="regression-approach-for-quantitative-traits" class="level2">
<h2 class="anchored" data-anchor-id="regression-approach-for-quantitative-traits">Regression Approach for Quantitative Traits</h2>
<ul>
<li><strong>Quantitative trait</strong>: e.g., height, BMI, systolic blood pressure</li>
<li><strong>Regression model</strong>: Y = β0 + β1 ⋅ X1 + ε
<ul>
<li>ε ∼ N(0, σ2)</li>
<li>Linear Regression</li>
</ul></li>
<li><strong>Genotype</strong>: aa, aA, AA</li>
<li><strong>Dosage (X1)</strong>: number of A alleles</li>
</ul>
<hr>
</section>
<section id="regression-approach-for-disease-traits" class="level2">
<h2 class="anchored" data-anchor-id="regression-approach-for-disease-traits">Regression Approach for Disease Traits</h2>
<ul>
<li><strong>Binary trait</strong>: e.g., disease status, hypertension</li>
<li><strong>Regression model</strong>: Y ~ Bernoulli(π)
<ul>
<li>logit(π) = log(π / (1 - π)) = β0 + β1 ⋅ X1</li>
<li>P(Y = 1) = π</li>
<li><strong>Odds</strong>: π / (1 - π)</li>
</ul></li>
<li><strong>Genotype</strong>: aa, aA, AA</li>
<li><strong>Dosage (X1)</strong>: number of A alleles</li>
<li><strong>Logistic Regression</strong>: beta1 = log odds ratio</li>
</ul>
<hr>
</section>
<section id="genome-wide-association-studies-gwas" class="level2">
<h2 class="anchored" data-anchor-id="genome-wide-association-studies-gwas">Genome-wide Association Studies (GWAS)</h2>
<ul>
<li>GWAS collect individuals (cases and controls for disease or a population sample for quantitative traits), measures the genotype of individuals in a set number of genomic locations (~1 million markers), and performs association tests between the phenotype (case status or quantitative trait) and each of the genetic markers (typically SNPs).</li>
</ul>
<hr>
</section>
<section id="gwas-genotype-phenotype" class="level2">
<h2 class="anchored" data-anchor-id="gwas-genotype-phenotype">GWAS: Genotype + Phenotype</h2>
<ul>
<li>The genotype of the individuals is represented as a matrix Xl, and the phenotype is represented as a vector Y.</li>
<li>Single marker SNP association test is performed leading to a table of SNP-level results with effect size, standard error of the effect size, and p-values.</li>
</ul>
<hr>
</section>
<section id="manhattan-plot" class="level2">
<h2 class="anchored" data-anchor-id="manhattan-plot">Manhattan Plot</h2>
<ul>
<li>Manhattan plots are used to visualize GWAS results. Points above -log(5 × 10^-8) are called GWAS significant.</li>
<li>GWAS significant threshold: 5e-8</li>
</ul>
<hr>
</section>
<section id="qq-plot" class="level2">
<h2 class="anchored" data-anchor-id="qq-plot">QQ Plot</h2>
<ul>
<li>A Q-Q plot is a useful tool to present the GWAS results and check for potential issues.</li>
<li>X-axis: the expected -log(P-values) under the null hypothesis of no association.</li>
<li>Y-axis: the observed -log(P-values).</li>
<li>Dots above the 45-degree line (upper right) deserve a closer look.</li>
</ul>
<hr>
</section>
<section id="gwas-snp-trait-discovery-timeline" class="level2">
<h2 class="anchored" data-anchor-id="gwas-snp-trait-discovery-timeline">GWAS SNP-Trait Discovery Timeline</h2>
<ul>
<li>GWAS have been so successful that thousands of them have been performed since 2005 with discoveries that grow continuously. As of 2020, more than 70K SNP-trait associations are reported in the GWAS catalog.</li>
</ul>
<hr>
</section>
<section id="wtccc-first-large-scale-gwas" class="level2">
<h2 class="anchored" data-anchor-id="wtccc-first-large-scale-gwas">WTCCC: First Large Scale GWAS</h2>
<ul>
<li>The Wellcome Trust Case Control Consortium’s (WTCCC) GWAS is a landmark study of 7 common diseases, the first large-scale GWAS performed.</li>
<li>GWA studies of 2000 cases and 3000 shared controls for 7 diseases</li>
<li>Platform: Affymetrix 500K Set</li>
<li>Main paper published in 2007</li>
<li>Results and summaries freely available</li>
<li>Genotype data access granted to qualified investigators</li>
</ul>
<hr>
</section>
<section id="wtccc-population-structure" class="level2">
<h2 class="anchored" data-anchor-id="wtccc-population-structure">WTCCC: Population Structure</h2>
<ul>
<li>QQ plots for WTCCC diseases show some departure from the identity line early on, indicating some population structure.</li>
<li>Methods for correcting for population structure were still underdeveloped at this time.</li>
</ul>
<hr>
</section>
<section id="zoom-in-with-regional-plots" class="level2">
<h2 class="anchored" data-anchor-id="zoom-in-with-regional-plots">Zoom in with Regional Plots</h2>
<ul>
<li>We can zoom into the Manhattan plot using locus zoom. These are staples of any GWAS paper at the moment.</li>
<li>Variants in LD tend to have similar association p-values. This is a good sign. If we find variants that are significant by themselves, this may be a sign of genotyping issue.</li>
</ul>
<hr>
</section>
<section id="clarifications-and-logistics" class="level2">
<h2 class="anchored" data-anchor-id="clarifications-and-logistics">Clarifications and Logistics</h2>
<ul>
<li>At the end of each lecture, you will have 5 minutes to post a summary of what you understood on canvas.</li>
<li>Homework problems are part of the learning process and may cover</li>
</ul>


<!-- -->

</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  const viewSource = window.document.getElementById('quarto-view-source') ||
                     window.document.getElementById('quarto-code-tools-source');
  if (viewSource) {
    const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
    viewSource.addEventListener("click", function(e) {
      if (sourceUrl) {
        // rstudio viewer pane
        if (/\bcapabilities=\b/.test(window.location)) {
          window.open(sourceUrl);
        } else {
          window.location.href = sourceUrl;
        }
      } else {
        const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
        modal.show();
      }
      return false;
    });
  }
  function toggleCodeHandler(show) {
    return function(e) {
      const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
      for (let i=0; i<detailsSrc.length; i++) {
        const details = detailsSrc[i].parentElement;
        if (show) {
          details.open = true;
        } else {
          details.removeAttribute("open");
        }
      }
      const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
      const fromCls = show ? "hidden" : "unhidden";
      const toCls = show ? "unhidden" : "hidden";
      for (let i=0; i<cellCodeDivs.length; i++) {
        const codeDiv = cellCodeDivs[i];
        if (codeDiv.classList.contains(fromCls)) {
          codeDiv.classList.remove(fromCls);
          codeDiv.classList.add(toCls);
        } 
      }
      return false;
    }
  }
  const hideAllCode = window.document.getElementById("quarto-hide-all-code");
  if (hideAllCode) {
    hideAllCode.addEventListener("click", toggleCodeHandler(false));
  }
  const showAllCode = window.document.getElementById("quarto-show-all-code");
  if (showAllCode) {
    showAllCode.addEventListener("click", toggleCodeHandler(true));
  }
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script><div class="modal fade" id="quarto-embedded-source-code-modal" tabindex="-1" aria-labelledby="quarto-embedded-source-code-modal-label" aria-hidden="true"><div class="modal-dialog modal-dialog-scrollable"><div class="modal-content"><div class="modal-header"><h5 class="modal-title" id="quarto-embedded-source-code-modal-label">Source Code</h5><button class="btn-close" data-bs-dismiss="modal"></button></div><div class="modal-body"><div class="">
<div class="sourceCode" id="cb1" data-shortcodes="false"><pre class="sourceCode markdown code-with-copy"><code class="sourceCode markdown"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> "Dissection of Genetics &amp; Prediction of Cancer Risk: Methods &amp; Tools"</span></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="an">author:</span><span class="co"> "Haky Im"</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="an">date:</span><span class="co"> "March 20, 2024"</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="an">format:</span><span class="co"> </span></span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="co">  revealjs: </span></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="co">    self-contained: true</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="an">draft:</span><span class="co"> true</span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="fu">## Goals of the Class</span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Learn concepts and tools needed to understand the current literature in cancer genomics and interpret the results</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>First month: focus on germline risk factors</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Today:</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Gene mapping: Linkage vs. association approaches</span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Describe the genetic architecture of cancer risk</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Discuss Genome-wide Association Studies (GWAS)</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="fu">## Linkage Analysis</span></span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Gene mapping method popular before development of high throughput genotyping techniques</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>100's of markers interrogated (causal markers most likely missed)</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Important genes were found with this approach</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>CFTR: Cystic Fibrosis</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>HTT: Huntington disease</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>NOD2: inflammatory bowel disease</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Markers that are co-transmitted (recombination rate&lt;0.5) with the disease susceptibility locus (DSL) help locate the DSL</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>It uses families with many cases ("loaded pedigrees")</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a><span class="fu">## High Penetrance Cancer Susceptibility Genes</span></span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Genetic linkage performed in the 1980s and 1990s found:</span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Breast and ovarian cancers</span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>BRCA1</span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>BRCA2</span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Colorectal cancer</span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>adenomatous polyposis coli (APC)</span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>mismatch repair genes</span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>mutL homologue 1 (MLH1)</span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>mutS homologue 2 (MSH2)</span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Melanoma: cyclin-dependent kinase inhibitor 2A (CDKN2A)</span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>To date mutations in more than 70 CSGs associated with high-penetrance cancer susceptibility syndromes have been identified with RRs of 5-100</span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-50"><a href="#cb1-50" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-51"><a href="#cb1-51" aria-hidden="true" tabindex="-1"></a><span class="fu">## 1996 - Power of Association vs Linkage</span></span>
<span id="cb1-52"><a href="#cb1-52" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-53"><a href="#cb1-53" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Risch N. &amp; Merikangas K. The future of genetic studies of complex human diseases. Science 273, 1516–1517 (1996).</span>
<span id="cb1-54"><a href="#cb1-54" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Many high penetrance (large effect) cancer susceptibility genes were found using linkage analysis.</span>
<span id="cb1-55"><a href="#cb1-55" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>By mid 90's discoveries using linkage were dwindling prompting the influential paper by Neil Risch and Merikangas to propose a revolutionary idea of genotyping about 100K loci in the genome (considered in the realm of science fiction at the time).</span>
<span id="cb1-56"><a href="#cb1-56" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Technological advances brought about by the human genome project made genome-wide association studies possible.</span>
<span id="cb1-57"><a href="#cb1-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-58"><a href="#cb1-58" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-59"><a href="#cb1-59" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-60"><a href="#cb1-60" aria-hidden="true" tabindex="-1"></a><span class="fu">## Gene Mapping Methods</span></span>
<span id="cb1-61"><a href="#cb1-61" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-62"><a href="#cb1-62" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Linkage analysis**</span>
<span id="cb1-63"><a href="#cb1-63" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>popular before high throughput technology became available</span>
<span id="cb1-64"><a href="#cb1-64" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>based on co-transmission of genetic markers and disease genes</span>
<span id="cb1-65"><a href="#cb1-65" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>a few hundred markers can cover the whole genome</span>
<span id="cb1-66"><a href="#cb1-66" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>low resolution</span>
<span id="cb1-67"><a href="#cb1-67" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>pedigree/family based</span>
<span id="cb1-68"><a href="#cb1-68" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Association mapping**</span>
<span id="cb1-69"><a href="#cb1-69" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>need a large number of markers (&gt; 1 million)</span>
<span id="cb1-70"><a href="#cb1-70" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>genetic markers in LD (Linkage Disequilibrium) with disease genes</span>
<span id="cb1-71"><a href="#cb1-71" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>higher resolution</span>
<span id="cb1-72"><a href="#cb1-72" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>families not needed (easier to recruit large cohorts)</span>
<span id="cb1-73"><a href="#cb1-73" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-74"><a href="#cb1-74" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-75"><a href="#cb1-75" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-76"><a href="#cb1-76" aria-hidden="true" tabindex="-1"></a><span class="fu">## Expected to Find High Penetrance Genes</span></span>
<span id="cb1-77"><a href="#cb1-77" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-78"><a href="#cb1-78" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Recap:</span>
<span id="cb1-79"><a href="#cb1-79" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Linkage and association are main approaches for mapping genes to diseases and other human traits.</span>
<span id="cb1-80"><a href="#cb1-80" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Before the completion of the Human Genome Project, linkage analysis was the most popular approach to identify disease loci.</span>
<span id="cb1-81"><a href="#cb1-81" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Association methods are based on the LD between genetic markers and disease genes.</span>
<span id="cb1-82"><a href="#cb1-82" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>We've been performing genome-wide association studies for the last two decades, expecting to find high penetrance (large effect size) genes.</span>
<span id="cb1-83"><a href="#cb1-83" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-84"><a href="#cb1-84" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-85"><a href="#cb1-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-86"><a href="#cb1-86" aria-hidden="true" tabindex="-1"></a><span class="fu">## Found Many Low Penetrance Variants</span></span>
<span id="cb1-87"><a href="#cb1-87" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-88"><a href="#cb1-88" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Instead, we found a large number of variants with modest effect sizes (low penetrance).</span>
<span id="cb1-89"><a href="#cb1-89" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We found that the genetic architecture of cancer risk was highly polygenic.</span>
<span id="cb1-90"><a href="#cb1-90" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>This requires better statistical methods to make sense of the discoveries.</span>
<span id="cb1-91"><a href="#cb1-91" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-92"><a href="#cb1-92" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-93"><a href="#cb1-93" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-94"><a href="#cb1-94" aria-hidden="true" tabindex="-1"></a><span class="fu">## Effect Size vs. Allele Frequency</span></span>
<span id="cb1-95"><a href="#cb1-95" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-96"><a href="#cb1-96" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>The figure shows that the discoverability of the risk alleles depends on the population frequency.</span>
<span id="cb1-97"><a href="#cb1-97" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>GWAS have unlocked many discoveries on the lower left, but to make sense of them and turn them into actionable targets, we need to learn new statistical concepts.</span>
<span id="cb1-98"><a href="#cb1-98" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Low penetrance = low relative risk = small effect size</span>
<span id="cb1-99"><a href="#cb1-99" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-100"><a href="#cb1-100" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-101"><a href="#cb1-101" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-102"><a href="#cb1-102" aria-hidden="true" tabindex="-1"></a><span class="fu">## Review: Hypothesis Testing</span></span>
<span id="cb1-103"><a href="#cb1-103" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-104"><a href="#cb1-104" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**H0**: null hypothesis (e.g., no difference in case-control allele frequencies)</span>
<span id="cb1-105"><a href="#cb1-105" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**HA**: alternative hypothesis (there is a difference, i.e., correlated with the causal variant)</span>
<span id="cb1-106"><a href="#cb1-106" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Test statistic Z**: in many situations associated with a model</span>
<span id="cb1-107"><a href="#cb1-107" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Significance level (α)**: allowed type I error (reject null when null is true)</span>
<span id="cb1-108"><a href="#cb1-108" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**p-value**: P(observed test statistic more extreme than the threshold | H0)</span>
<span id="cb1-109"><a href="#cb1-109" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We reject the null hypothesis if the observed test statistic is more extreme than what we would expect could happen by chance.</span>
<span id="cb1-110"><a href="#cb1-110" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-111"><a href="#cb1-111" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-112"><a href="#cb1-112" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-113"><a href="#cb1-113" aria-hidden="true" tabindex="-1"></a><span class="fu">## Review: Hypothesis Testing (Cont.)</span></span>
<span id="cb1-114"><a href="#cb1-114" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-115"><a href="#cb1-115" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Type I error (α)**: probability of rejecting the null when the null is true</span>
<span id="cb1-116"><a href="#cb1-116" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Type II error (β)**: probability of not rejecting the null when the null is false</span>
<span id="cb1-117"><a href="#cb1-117" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Power**: probability that we will reject the null when the alternative is true</span>
<span id="cb1-118"><a href="#cb1-118" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**P-value**: probability that we will get a result as extreme or more extreme than the observed data point under the null hypothesis</span>
<span id="cb1-119"><a href="#cb1-119" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-120"><a href="#cb1-120" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-121"><a href="#cb1-121" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-122"><a href="#cb1-122" aria-hidden="true" tabindex="-1"></a><span class="fu">## Regression Approach: Single SNP</span></span>
<span id="cb1-123"><a href="#cb1-123" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-124"><a href="#cb1-124" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>SNP: single nucleotide polymorphism</span>
<span id="cb1-125"><a href="#cb1-125" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Regression model**: Y = μ + a ⋅ age + β ⋅ X + ε</span>
<span id="cb1-126"><a href="#cb1-126" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Parameters β are estimated (using MLE, least squares, etc.)</span>
<span id="cb1-127"><a href="#cb1-127" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Null hypothesis: β = 0</span>
<span id="cb1-128"><a href="#cb1-128" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Many types of traits can be treated with the same approach</span>
<span id="cb1-129"><a href="#cb1-129" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Can correct for covariates (age, sex, ancestry)</span>
<span id="cb1-130"><a href="#cb1-130" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Prediction</span>
<span id="cb1-131"><a href="#cb1-131" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Genotype**: aa, aA, AA</span>
<span id="cb1-132"><a href="#cb1-132" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Dosage (X1)**: number of A alleles</span>
<span id="cb1-133"><a href="#cb1-133" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-134"><a href="#cb1-134" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-135"><a href="#cb1-135" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-136"><a href="#cb1-136" aria-hidden="true" tabindex="-1"></a><span class="fu">## Regression Approach for Quantitative Traits</span></span>
<span id="cb1-137"><a href="#cb1-137" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-138"><a href="#cb1-138" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Quantitative trait**: e.g., height, BMI, systolic blood pressure</span>
<span id="cb1-139"><a href="#cb1-139" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Regression model**: Y = β0 + β1 ⋅ X1 + ε</span>
<span id="cb1-140"><a href="#cb1-140" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>ε ∼ N(0, σ2)</span>
<span id="cb1-141"><a href="#cb1-141" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>Linear Regression</span>
<span id="cb1-142"><a href="#cb1-142" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Genotype**: aa, aA, AA</span>
<span id="cb1-143"><a href="#cb1-143" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Dosage (X1)**: number of A alleles</span>
<span id="cb1-144"><a href="#cb1-144" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-145"><a href="#cb1-145" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-146"><a href="#cb1-146" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-147"><a href="#cb1-147" aria-hidden="true" tabindex="-1"></a><span class="fu">## Regression Approach for Disease Traits</span></span>
<span id="cb1-148"><a href="#cb1-148" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-149"><a href="#cb1-149" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Binary trait**: e.g., disease status, hypertension</span>
<span id="cb1-150"><a href="#cb1-150" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Regression model**: Y ~ Bernoulli(π)</span>
<span id="cb1-151"><a href="#cb1-151" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>logit(π) = log(π / (1 - π)) = β0 + β1 ⋅ X1</span>
<span id="cb1-152"><a href="#cb1-152" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>P(Y = 1) = π</span>
<span id="cb1-153"><a href="#cb1-153" aria-hidden="true" tabindex="-1"></a><span class="ss">  - </span>**Odds**: π / (1 - π)</span>
<span id="cb1-154"><a href="#cb1-154" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Genotype**: aa, aA, AA</span>
<span id="cb1-155"><a href="#cb1-155" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Dosage (X1)**: number of A alleles</span>
<span id="cb1-156"><a href="#cb1-156" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Logistic Regression**: beta1 = log odds ratio</span>
<span id="cb1-157"><a href="#cb1-157" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-158"><a href="#cb1-158" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-159"><a href="#cb1-159" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-160"><a href="#cb1-160" aria-hidden="true" tabindex="-1"></a><span class="fu">## Genome-wide Association Studies (GWAS)</span></span>
<span id="cb1-161"><a href="#cb1-161" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-162"><a href="#cb1-162" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>GWAS collect individuals (cases and controls for disease or a population sample for quantitative traits), measures the genotype of individuals in a set number of genomic locations (~1 million markers), and performs association tests between the phenotype (case status or quantitative trait) and each of the genetic markers (typically SNPs).</span>
<span id="cb1-163"><a href="#cb1-163" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-164"><a href="#cb1-164" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-165"><a href="#cb1-165" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-166"><a href="#cb1-166" aria-hidden="true" tabindex="-1"></a><span class="fu">## GWAS: Genotype + Phenotype</span></span>
<span id="cb1-167"><a href="#cb1-167" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-168"><a href="#cb1-168" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>The genotype of the individuals is represented as a matrix Xl, and the phenotype is represented as a vector Y.</span>
<span id="cb1-169"><a href="#cb1-169" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Single marker SNP association test is performed leading to a table of SNP-level results with effect size, standard error of the effect size, and p-values.</span>
<span id="cb1-170"><a href="#cb1-170" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-171"><a href="#cb1-171" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-172"><a href="#cb1-172" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-173"><a href="#cb1-173" aria-hidden="true" tabindex="-1"></a><span class="fu">## Manhattan Plot</span></span>
<span id="cb1-174"><a href="#cb1-174" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-175"><a href="#cb1-175" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Manhattan plots are used to visualize GWAS results. Points above -log(5 × 10^-8) are called GWAS significant.</span>
<span id="cb1-176"><a href="#cb1-176" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>GWAS significant threshold: 5e-8</span>
<span id="cb1-177"><a href="#cb1-177" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-178"><a href="#cb1-178" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-179"><a href="#cb1-179" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-180"><a href="#cb1-180" aria-hidden="true" tabindex="-1"></a><span class="fu">## QQ Plot</span></span>
<span id="cb1-181"><a href="#cb1-181" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-182"><a href="#cb1-182" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>A Q-Q plot is a useful tool to present the GWAS results and check for potential issues.</span>
<span id="cb1-183"><a href="#cb1-183" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>X-axis: the expected -log(P-values) under the null hypothesis of no association.</span>
<span id="cb1-184"><a href="#cb1-184" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Y-axis: the observed -log(P-values).</span>
<span id="cb1-185"><a href="#cb1-185" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Dots above the 45-degree line (upper right) deserve a closer look.</span>
<span id="cb1-186"><a href="#cb1-186" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-187"><a href="#cb1-187" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-188"><a href="#cb1-188" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-189"><a href="#cb1-189" aria-hidden="true" tabindex="-1"></a><span class="fu">## GWAS SNP-Trait Discovery Timeline</span></span>
<span id="cb1-190"><a href="#cb1-190" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-191"><a href="#cb1-191" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>GWAS have been so successful that thousands of them have been performed since 2005 with discoveries that grow continuously. As of 2020, more than 70K SNP-trait associations are reported in the GWAS catalog.</span>
<span id="cb1-192"><a href="#cb1-192" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-193"><a href="#cb1-193" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-194"><a href="#cb1-194" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-195"><a href="#cb1-195" aria-hidden="true" tabindex="-1"></a><span class="fu">## WTCCC: First Large Scale GWAS</span></span>
<span id="cb1-196"><a href="#cb1-196" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-197"><a href="#cb1-197" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>The Wellcome Trust Case Control Consortium's (WTCCC) GWAS is a landmark study of 7 common diseases, the first large-scale GWAS performed.</span>
<span id="cb1-198"><a href="#cb1-198" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>GWA studies of 2000 cases and 3000 shared controls for 7 diseases</span>
<span id="cb1-199"><a href="#cb1-199" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Platform: Affymetrix 500K Set</span>
<span id="cb1-200"><a href="#cb1-200" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Main paper published in 2007</span>
<span id="cb1-201"><a href="#cb1-201" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Results and summaries freely available</span>
<span id="cb1-202"><a href="#cb1-202" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Genotype data access granted to qualified investigators</span>
<span id="cb1-203"><a href="#cb1-203" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-204"><a href="#cb1-204" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-205"><a href="#cb1-205" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-206"><a href="#cb1-206" aria-hidden="true" tabindex="-1"></a><span class="fu">## WTCCC: Population Structure</span></span>
<span id="cb1-207"><a href="#cb1-207" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-208"><a href="#cb1-208" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>QQ plots for WTCCC diseases show some departure from the identity line early on, indicating some population structure.</span>
<span id="cb1-209"><a href="#cb1-209" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Methods for correcting for population structure were still underdeveloped at this time.</span>
<span id="cb1-210"><a href="#cb1-210" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-211"><a href="#cb1-211" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-212"><a href="#cb1-212" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-213"><a href="#cb1-213" aria-hidden="true" tabindex="-1"></a><span class="fu">## Zoom in with Regional Plots</span></span>
<span id="cb1-214"><a href="#cb1-214" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-215"><a href="#cb1-215" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We can zoom into the Manhattan plot using locus zoom. These are staples of any GWAS paper at the moment.</span>
<span id="cb1-216"><a href="#cb1-216" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Variants in LD tend to have similar association p-values. This is a good sign. If we find variants that are significant by themselves, this may be a sign of genotyping issue.</span>
<span id="cb1-217"><a href="#cb1-217" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-218"><a href="#cb1-218" aria-hidden="true" tabindex="-1"></a>---</span>
<span id="cb1-219"><a href="#cb1-219" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-220"><a href="#cb1-220" aria-hidden="true" tabindex="-1"></a><span class="fu">## Clarifications and Logistics</span></span>
<span id="cb1-221"><a href="#cb1-221" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-222"><a href="#cb1-222" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>At the end of each lecture, you will have 5 minutes to post a summary of what you understood on canvas.</span>
<span id="cb1-223"><a href="#cb1-223" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Homework problems are part of the learning process and may cover</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div></div></div></div>
</div> <!-- /content -->



</body></html>